### PR TITLE
Auth config for TS and Java

### PR DESCRIPTION
**Pull request by efirs**

### CODE REVIEW

File: docs/java/getting-started.mdx [hunk 1]
```diff
@@ -62,15 +60,36 @@ maximize the use of system resources.
...
+<TabItem value="Cloud" label="Cloud">
 
-<Tabs groupId={clientTabGroupId} className={tabbedItemTheme}>
-<TabItem value={syncClientLabel} label={syncClientLabel}>
+Tigris URL, ClientID and ClientSecret need to be set as follows,
+in order to connect to the hosted platform:
+
+```java
+TigrisConfiguration configuration = TigrisConfiguration.newBuilder("localhost:8081")
+    .withAuthConfig(new TigrisConfiguration.AuthConfig("paste client_id here", "paste client_secret here"))
```
Comment (JigarJoshi) [inline docs/java/getting-started.mdx:71]: the URL needs fixed.

Comment (efirs) [reply docs/java/getting-started.mdx:71]: good catch!

File: docs/golang/getting-started.md [hunk 2]
```diff
@@ -46,15 +50,8 @@ The `OpenDatabase` function connects to the Tigris backend, creates the
...
-
-```go
-db, err := tigris.OpenDatabase(ctx,
-	&config.Database{Driver: config.Driver{URL: "localhost:8081"}},
-    "catalogdb", &Catalog{})
-```
-
-#### Cloud platform configuration
+<Tabs className={tabbedItemTheme}>
+<TabItem value="Cloud" label="Cloud">
```
Comment (ovaistariq) [inline docs/golang/getting-started.md:54]: Tigris Cloud

File: docs/golang/getting-started.md [hunk 3]
```diff
@@ -73,3 +70,15 @@ db, err := tigris.OpenDatabase(ctx, cfg, "catalogdb", &Catalog{})
 
 The ClientID and ClientSecret can be retrieved by creating an application
 in the [CLI](../cli/authentication.md#application-credentials) or UI.
+
+</TabItem>
+<TabItem value="Development" label="Development">
```
Comment (ovaistariq) [inline docs/golang/getting-started.md:75]: Development Environment

File: docs/golang/getting-started.md [hunk 4]
```diff
@@ -73,3 +70,15 @@ db, err := tigris.OpenDatabase(ctx, cfg, "catalogdb", &Catalog{})
...
+
+```go
+db, err := tigris.OpenDatabase(ctx,
```
Comment (ovaistariq) [inline docs/golang/getting-started.md:78]: You had added another method that explicitly connects. Can we change the example to use that? Then we can make the order of the content better, by first showing connection, then showing the models and finally showing how to create the database and collections.

Unfortunately while OpenDatabase provides great abstraction, at this point where users want to learn more about what's happening it is hiding the process.

File: docs/java/getting-started.mdx [hunk 5]
```diff
@@ -62,15 +60,36 @@ maximize the use of system resources.
 
 ### Initialize
```
Comment (ovaistariq) [inline docs/java/getting-started.mdx:61]: Can you change this to `Create Connection`

File: docs/typescript/getting-started.md [hunk 6]
```diff
@@ -80,14 +84,39 @@ For detailed documentation on schema representation refer to the
 
 ## Connect and initialize the database
```
Comment (ovaistariq) [inline docs/typescript/getting-started.md:85]: Could you re-order this section as well:

- Installation
- Create connection / Connect
- Setup the data models
- Create database and collections

File: docs/golang/getting-started.md [hunk 7]
```diff
@@ -8,13 +12,47 @@ or newer.
...
+<Tabs className={tabbedItemTheme}>
+<TabItem value="Tigris Cloud" label="Tigris Cloud">
+
+Tigris URL, ClientID and ClientSecret need to be set as follows,
+in order to connect to the hosted platform:
+
+```go
+cfg := &config.Database{
+	Driver: config.Driver{
+		URL: "api.preview.tigrisdata.cloud"
```
Comment (JigarJoshi) [inline docs/golang/getting-started.md:32]: URL: "api.preview.tigrisdata.cloud:443"

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**